src/comment.rs

```diff
@@ -579,7 +579,11 @@ fn rewrite_comment_inner(
             let item_fmt = ib.create_string_format(&fmt);
             result.push_str(&comment_line_separator);
             result.push_str(&ib.opener);
-            match rewrite_string(&item_block_buffer.replace("\n", " "), &item_fmt) {
+            match rewrite_string(
+                &item_block_buffer.replace("\n", " "),
+                &item_fmt,
+                max_chars - ib.indent,
```

Comment:
Would you use `saturating_sub` here (and in other places as well) to avoid panicking on extreme corner cases please?

Reply:
makes sense, thanks!